server/intelligence-service/app/model.py

```diff
@@ -60,7 +61,7 @@ class State(TypedDict):
     token_counter=model,
     include_system=True,
     allow_partial=False,
-    start_on="human",
+    start_on="human",  # TODO: change to "system"
```

Comment:
Is this TODO meant for this PR? Otherwise maybe create and link a follow-up issue, so that TODOs don't stay in here forever.